Sources/Build/BuildOperationBuildSystemDelegateHandler.swift

```diff
@@ -619,6 +619,14 @@ final class BuildOperationBuildSystemDelegateHandler: LLBuildBuildSystemDelegate
         self.commandFailureHandler?()
     }
 
+    func buildStart(configuration: BuildConfiguration) {
+        queue.sync {
+            self.progressAnimation.clear()
+            self.outputStream <<< "Building for \(configuration == .debug ? "debugging" : "production")...\n"
```

Comment:
Nit: should we say "release" to be consistent with the name of the config?

Reply:
Yeah, maybe indeed more consistent 🤔  The "debugging" one was the important one I really wanted;
Should we follow up with a rename of production -> release?